### PR TITLE
Minor improvements to formatting, spelling, and flow.

### PR DESCRIPTION
**Pull request by dideler**
Just a little quality control. No new information has been added (I've never even ~~worked~~ trained in the US).

You can cherry pick my commits if you don't agree with them all.

### CODE REVIEW

**Comment by dideler** [comment]
Don't merge anything yet. I just checked http://dennisideler.com/Unofficial-Waterloo-USA-Intern-Guide/ and there seems to be some error (possibly related to parsing), the document isn't showing up correctly. I probably screwed up somewhere in one of my commits. I'll look into it tomorrow, time to sleep.

File: index.markdown [hunk 1]
```diff
@@ -435,7 +446,7 @@ Spreads are not always fixed, weekends can cost more so be careful.
 <section id="Money-Taxes" />
 ### Taxes
 *Disclaimer*: No author of this document knows anything about taxes, accounting
-or even arithmetic. Do not rely on this document. Speak to a professional.
+or, even arithmetic. Do not rely on this document. Speak to a professional.
```
Comment (sholiday) [inline index.markdown]: Is this comma proper? I'm no grammar expert but it reads odd to me.

**Comment by sholiday** [comment]
Looks, good (just that comma). Thanks for the cleanup?

How did you find this and what interested you to clean it up?

File: index.markdown (hunk 1 above)
Comment (dideler) [inline index.markdown]: Good catch. I meant to put the comma before 'or'. The original statement reminded me of this

**Comment by dideler** [comment]
I found it by someone linking to it on reddit. As I read over it I noticed lots of minor things that could be improved, so I figured why not.

**Comment by sholiday** [comment]
Completely forgot about this! Thanks again for your contributions!

**Comment by dideler** [comment]
No problem, glad to help.

**Pull request merged**